tidb_test.go

```diff
@@ -832,6 +832,21 @@ func (s *testSessionSuite) TestShow(c *C) {
 	match(c, row, "utf8_bin", "utf8", 83, "", "Yes", 1)
 }
 
+func (s *testSessionSuite) TestTimeFunc(c *C) {
+	store := newStore(c, s.dbName)
+	se := newSession(c, store, s.dbName)
+
+	last := time.Now().Format(mysql.TimeFormat)
+	r := mustExecSQL(c, se, "select now(), now(6), current_timestamp, current_timestamp(), current_timestamp(6)")
```

Comment:
current_timestamp is synonymy for now, they are the same. 

btw, I don't check equal, I check greater or equal last timestamp before current execution.